tests/phpunit/elementor/core/logger/test-manager.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Elementor\Tests\Phpunit\Elementor\Core\Logger;
+
+use ElementorEditorTesting\Elementor_Test_Base;
+
+class Test_Manager extends Elementor_Test_Base {
+
+	/**
+	 * @var \Elementor\Core\Logger\Manager
+	 */
+	private $manager;
+
+	public function setUp() {
+		$this->manager = $this->getMockBuilder( \Elementor\Core\Logger\Manager::class )
+		                      ->setMethods( [ 'shutdown' ] )
+		                      ->getMock();
+
+		$this->manager->method( 'shutdown' )
+		              ->willReturn( true );
```

Comment:
```suggestion
		// Mock 'shutdown' method to avoid exit.
		$this->manager = $this->getMockBuilder( \Elementor\Core\Logger\Manager::class )
		                      ->setMethods( [ 'shutdown' ] )
		                      ->getMock();

		$this->manager->method( 'shutdown' )
		              ->willReturn( true );
```